docs/source/docker/getting_started/first_steps.rst

```diff
@@ -530,6 +530,52 @@ It is also possible to directly download the actual files themselves as follows:
     )
 
 
+Sharing Datasets
+----------------
+
+Once a dataset has been created we can also make it accessible to other users by
+sharing it. Sharing works through e-mail addresses. 
+
+.. code-block:: python
+    :caption: Share a dataset
+
+    # we first need to have an api client (create a new or use an existing one)
+    client = ApiWorkflowClient(token="MY_AWESOME_TOKEN")
+
+    # share a dataset with an user
+    client.share_dataset_only_with(dataset_id="MY_DATASET_ID", user_emails=["user@something.com"])
+
+    # share dataset with a user while keep sharing it with previous users
+    user_emails = client.get_shared_users(dataset_id="MY_DATASET_ID")
+    user_emails.extend(["additional_user2@something.com"])
+    client.share_dataset_only_with(dataset_id="MY_DATASET_ID", user_emails=user_emails)
+
+    # revoke access to all users
+    client.share_dataset_only_with(dataset_id="MY_DATASET_ID", user_emails=[])
+
+
+If you want to get a list of users that have access to a given dataset we can do 
+this using the following code: 
+
+.. code-block:: python
+    :caption: Share a dataset
+
+    # we first need to have an api client (create a new or use an existing one)
+    client = ApiWorkflowClient(token="MY_AWESOME_TOKEN")
+
+    # get a list of users that have access to a given dataset
+    client.get_shared_users(dataset_id="MY_DATASET_ID")
+    # ["user@something.com"]
+
+You find more details about the individual API requests in the docs for the
+Python client: :py:class:`lightly.api.api_workflow_client`
```

Comment:
I am not sure why this is necessary. Do you want to linkt to all API requests in general?

Alternatively you can also link to specific methods:
`:py:meth:`get_shared_users <lightly.api.api_workflow_client.get_shared_users>``

Reply:
Yes, you're right. Let's drop it.